test/app/newApi/sendVTTs.spec.ts

```diff
@@ -0,0 +1,34 @@
+import * as api from "../../../app/renderer/new_api"
+
+describe("Import Seed method", () => {
+  it("should request Import method and return value", async () => {
+    const params = {
+      walletId: "Int",
+      toAddress: [],
```

Comment:
This will not work as a dummy address. The type of this field is expected to be `Buffer`, not `Array`. 

```
test/app/newApi/sendVTTs.spec.ts:29:50 - error TS2345: Argument of type '{ walletId: string; toAddress: never[]; amount: number; fee: number; subject: string; }' is not assignable to parameter of type 'TypeOfProps<{ walletId: StringType; toAddress: Type<Buffer, string, mixed>; amount: NumberType; fee: NumberType; subject: StringType; }>'.
  Types of property 'toAddress' are incompatible.
    Type 'never[]' is missing the following properties from type 'Buffer': write, toJSON, equals, compare, and 47 more.
29     const result = await api.sendVTT(mockClient, params)
                                                    ~~~~~~
```

Please replace the empty brackets `[]` by a proper `Buffer` initializator like `new Buffer([])`.

Reply:
Done!

Reply:
I replaced to Buffer.from([]) because new Buffer is deprecated now